apps/arena/lib/arena/game/player.ex

```diff
@@ -314,6 +314,11 @@ defmodule Arena.Game.Player do
     |> Enum.uniq()
   end
 
+  def apply_on_death_effect(player, effect, effect_id) do
```

Comment:
Shouldn't be `on_kill`? Since the given player is the killer (who receives the effect).

Reply:
Honestly it should just be `apply_effect` but I didn't want to deal the clash with the other functions... I'll rename to `put_effect`

Reply:
Done https://github.com/lambdaclass/mirra_backend/pull/389#:~:text=to%20put_effect/3-,85e4be6,-Refactor%20handle_info(%3Ato_killfeed